### PR TITLE
Kyung removed gtk requirement

### PR DESCRIPTION
**Pull request by Jothle12**
Pygtk was not in use, but was required for startup before. Now it isn't.

### CODE REVIEW

File: Main.py [hunk 1]
```diff
@@ -3,7 +3,10 @@
 import pygame
 import os
 import json
-from gi.repository import Gtk
+#from gi.repository import Gtk
```
Comment (Nolski) [inline Main.py:7]: Everything looks good. Is there any reason why this one is commented out and not just removed?

**Comment by lcb931023** [comment]
Lads and gents, I present to you......

Fractionauts 2.0!
cover-photo

Or shall we say, Fractionauts 4/2? XD

(Sorry about the obviously non-related name. We wrote a pull request ages ago, and it seems Github doesn't feel like allowing us more pull requests till this one is merged.)

**Comment by LanceALaughlin** [comment]
@lcb931023 You should look into gitflow to mitigate the issue with Github only allowing one pull request.

**Comment by lcb931023** [comment]
Will do, will do.
Can we merge this one? The quicker the better, to avoid more stacked commits.

**Comment by LanceALaughlin** [comment]
Merged!

**Pull request merged**